### PR TITLE
feat(presets): timed presets give Renovate 4 hours to create branches

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Presets that have a "time window" now have 4 hours to create branches

## Context

This change was suggested by @rarkins in this comment:

- https://github.com/renovatebot/renovate/discussions/22335#discussioncomment-5966868

We have presets that rely on (extend) the `schedule:earlyMondays` or `schedule:monthly` presets. Do you want me to label this a breaking change, and target our `v36` branch, so we merge it as a breaking change?

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/config/presets/internal/schedule.ts
Comment (HonkingGoose) [inline lib/config/presets/internal/schedule.ts:5]: The `daily` schedule should probably also get two hours extra?

**Review by rarkins — CHANGES_REQUESTED** [review]
Let's aim for 4 hours for each window

File: docs/usage/configuration-options.md
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:1695]: The tests are okay after my changes. I expected things to break on me somehow.

It still feels too much like a "draft" for me. It was more of a copy terms into IDE and find/replace manually what seems right to change...

Please thoroughly review this PR before you mark it ready to merge.

File: lib/config/presets/internal/schedule.ts
Comment (HonkingGoose) [inline lib/config/presets/internal/schedule.ts:7]: Copy/pasting this from the PR body, so I know it's seen by you maintainers. 😉 

> We have presets that rely on (extend) the `schedule:earlyMondays` or `schedule:monthly` presets. Do you want me to label this a breaking change, and target our `v36` branch, so we merge it as a breaking change?

Comment (rarkins) [reply lib/config/presets/internal/schedule.ts:7]: I think this is non-breaking

**Review by rarkins — APPROVED** [review]

**Review by viceice — APPROVED** [review]

**Pull request merged**
